CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contributing to the Hedera SDK for JavaScript
```

Comment:
```suggestion
# Contributing to the Hedera SDK for C++
```

Reply:
Fixed in ad3ea037429a4f2cf9cadc72d920c2cf9d020b39.